File: Backend/create_dirs.py
import os

# Create the directories
lib_dir = r'c:\Users\Naman\OneDrive\Desktop\InterShield\lib'
api_dir = r'c:\Users\Naman\OneDrive\Desktop\InterShield\pages\api'

os.makedirs(lib_dir, exist_ok=True)
os.makedirs(api_dir, exist_ok=True)

print(f"Created: {lib_dir}")
print(f"Created: {api_dir}")

File: Backend/create_dirs.js
const fs = require('fs');
const path = require('path');

const dirs = [
  r'c:\Users\Naman\OneDrive\Desktop\InterShield\lib',
  r'c:\Users\Naman\OneDrive\Desktop\InterShield\pages\api'
];

const results = [];

// Try fs.mkdirSync with recursive option
try {
  for (const dir of dirs) {
    fs.mkdirSync(dir, { recursive: true });
    if (fs.existsSync(dir)) {
      results.push(`✓ Node.js fs.mkdirSync - SUCCESS: ${dir}`);
    } else {
      results.push(`✗ Node.js fs.mkdirSync - FAILED: ${dir}`);
    }
  }
} catch (e) {
  results.push(`✗ Node.js fs.mkdirSync - ERROR: ${e.message}`);
}

console.log(results.join('\n'));
